lib/gooddata/rest/client.rb

```diff
@@ -188,6 +188,10 @@ def project_is_accessible?(id)
         end
       end
 
+      def segments(domain_id)
+        GoodData::Segment.all({ domain: domain_id })
```

Comment:
User Domain.segments() instead introducing new method.

Reply:
yes we will use Domain.segments() instead of